modules/services/osmose.js

```diff
@@ -0,0 +1,355 @@
+import RBush from 'rbush';
```

Comment:
I didn't review this file closely… I'm assuming it works fine since the issues load correctly 😅

Reply:
Yeah pretty confident about everything in this file I've spent so much time looking at it 😄 I want to convert more of it to using Promises instead of callbacks (just did for the issue detail requests), but can do that separate from this PR.